components/ActionSheet/tests/unit/MDCActionSheetTestHelper.m

```diff
@@ -24,10 +24,8 @@ + (NSArray *)colorsToTest {
   UIColor *hsbColor = [UIColor colorWithHue:0.8f saturation:0.8f brightness:0.8f alpha:0.8f];
   UIColor *blackWithAlpha = [UIColor.greenColor colorWithAlphaComponent:0.8f];
   UIColor *black = UIColor.blackColor;
-  CIColor *ciColor = [[CIColor alloc] initWithColor:UIColor.blackColor];
```

Comment:
Oh is it because we are using CoreImage?
If so then we can just add the dep to the build target and import it.

Reply:
I can add the import

Reply:
I am missing `#import <CoreImage/CoreImage.h>`

Reply:
yes cool thanks.